<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CModelMappingModifyForm</class>
 <widget class="QFrame" name="CModelMappingModifyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>237</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify model data</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gl_ModelMappingModifyForm">
   <item row="2" column="2">
    <widget class="QCheckBox" name="cb_Simulator">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="BlackGui::Components::CSimulatorSelector" name="frp_SimulatorSelector">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbl_Description">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>le_Description</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cb_Name">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lbl_Name">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>le_Name</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="le_Name">
     <property name="placeholderText">
      <string>Name</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="cb_Description">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbl_Simulator">
     <property name="text">
      <string>Simulator:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="le_Description">
     <property name="maxLength">
      <number>256</number>
     </property>
     <property name="placeholderText">
      <string>Model description</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lbl_Mode">
     <property name="text">
      <string>Mode:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="BlackGui::Components::CModelModeSelector" name="frp_IncludeSelector">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QCheckBox" name="cb_Mode">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CModelModeSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/modelmodeselector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
